sources/packages/forms/src/form-definitions/studentrequestchange.json

```diff
@@ -455,6 +455,10 @@
               {
                 "value": "studentdisabilityappeal",
                 "label": "Disability Information"
+              },
+              {
+                "value": "studentdependantsappealparttime",
+                "label": "Add/Remove Dependants (Part-Time Applications)"
```

Comment:
Minor. Also grammatically `Dependant` is valid but `Dependants`(plural) is not valid. 
So it could be called Dependents. Please confirm with business.

Reply:
Sure. Will check with Huge.

Reply:
I have a response from the business. @Joshua-Lakusta points out that the British way is the way we are supposed to follow. Auto correct wants "Dependent", but we should be using "Dependant" and "Dependants" as its plural in Canada. We use "Dependant" as the noun and "Dependent" as the adjective.  For example, we are dependent on our developers to do the development work, and my child is my dependant. Thanks @Joshua-Lakusta. I think we will keep using "dependant" and its plural "dependants" throughout the project.